lte/gateway/c/session_manager/EnumToString.cpp

```diff
@@ -61,6 +61,8 @@ std::string final_action_to_str(ChargingCredit_FinalAction final_action) {
 
 std::string grant_type_to_str(GrantTrackingType grant_type) {
   switch (grant_type) {
+    case ALL_TOTAL_TX_RX:
+      return "ALL_TOTAL_TX_RX (Total, Rx, Tx)";
```

Comment:
Not sure if we need the ` (Total, Rx, Tx)` :)

Reply:
I just meant that `ALL_TOTAL_TX_RX` impllies `(Total, Rx, Tx)` so we probably don't need the parenthesis